docs/index.md

```diff
@@ -0,0 +1,52 @@
+# FAIRmat nyaml documentation
```

Comment:
Wouldnt mention FAIRmat here but in the footer where indeed we should acknowledge the funding of the DFG

See an example here:
https://fairmat-nfdi.github.io/pynxtools-em/
bottom of the page with hyperlink to DFG page

Reply:
Maybe also a statement supported by the NIAC is useful